### DISCUSSION TITLE
Low memory available, stability problems may occur.

### DISCUSSION THREAD

**Discussion by YokaiKatana** [sanni/cartreader#346] [Coding Corner]
Hi, i got everything built and just putting the sketch in the Mega 2560 Pro,
nere the end of the upload i get this message:

Low memory available, stability problems may occur.

is this ok. or am i doing somthing wrong?
thanks in advance
![Low memory available, stability problems may occur](https://user-images.githubusercontent.com/98910589/158245593-772bd7f1-89cf-4c0e-aa79-da2bb461a3aa.JPG)

**Comment by sanni** [comment]
It's normal. The current LCD library is too big. It needs to be changed to something more lightweight in the future.
